app/src/main/java/com/it3048/mobiledeviceproject/MainActivity.kt

```diff
@@ -72,5 +72,87 @@ class MainActivity : AppCompatActivity() {
             else -> super.onOptionsItemSelected(item)
         }
     }
+    //Gesture Listener (when you swipe in a direction on the phone screen)
```

Comment:
Gesture code. From https://www.youtube.com/watch?v=j1aydFEOEA0 (Brandon's Module 10).

Reply:
Is this used anywhere, or is it just a copy paste from somewhere else?

This appears to be an incomplete feature.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality.
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.